### DISCUSSION TITLE
Is there a way to share dependency between projects?

### DISCUSSION THREAD

**Discussion by ilyachch** [python-poetry/poetry#7962] [Q&A]
In case, if we have several projects in monorepo:
```
.
├── project1
│   └── pyproject.toml
├── project2
│   └── pyproject.toml
├── project3
│   └── pyproject.toml
└── project4
    └── pyproject.toml
```
Is there any way to make some config to share between projects some of dependencies? Something like this:
```
.
├── project1
│   └── pyproject.toml
├── project2
│   └── pyproject.toml
├── project3
│   └── pyproject.toml
├── project4
│   └── pyproject.toml
└── root_pyproject.toml
```

with contents:
```
# root_pyproject.toml

[tool.poetry.dependencies]
framework = "1.0.1"

# project1/pyproject.toml
[tool.poetry.dependencies]
@import "../root_pyproject.toml"
```

Or anything similar by functions

**Comment by KPLauritzen** [comment]
Yes. See this example repo: https://github.com/KPLauritzen/python-monorepo

Basically, you can add "project1" as a dependency of "project2". 
https://github.com/KPLauritzen/python-monorepo/blob/95d8695bad890f910faae33669ae83634447f2af/src/service1/pyproject.toml#L9
